.github/workflows/ci.yml

```diff
@@ -85,13 +85,11 @@ jobs:
           tar --exclude=".git*" --exclude=target --exclude=third_party/prebuilt \
               -czvf target/release/deno_src.tar.gz -C .. deno
 
-      - uses: dtolnay/rust-toolchain@stable
-
-      - name: Install clippy and rustfmt
-        if: matrix.job == 'lint'
-        run: |
-          rustup component add clippy
-          rustup component add rustfmt
```

Comment:
We could remove this and put...

```toml
components = ["rustfmt", "clippy"]
```

...in rust-toolchain.toml in the root. Then only do:

```
- uses: dtolnay/rust-toolchain@stable
```